### ISSUE TITLE
Automated builds for ledger

### ISSUE THREAD

**Issue by AlexanderAA** [AlexanderAA/ledger_binaries_windows#5]

**Comment by fraga** [comment]
Hi @AlexanderAA just so you know travis has now windows support as OS in build script but it is alpha at the moment. I tried and for now wasn't successful. Check branch https://github.com/fraga/ledger-windows-build/tree/travisbuild. Any ideas?

**Comment by andrewsav-bt** [comment]
Most free cloud build environments have a cap on the time build can run. Boost takes a long to compile and usually go over the cap. If anyone has any idea how this could be resolved (using precompiled version of boost? From where?) Let me know.

Another thought, is that it would be nice if we could build it with a different toolchain, e.g. mingw, but if author of ledger-cli cannot support this, I doubt we can get far.

**Comment by fraga** [comment]
@andrewsav-datacom agree. Travis is currently building ledger in 15min or so https://travis-ci.org/ledger/ledger but I'm not sure if it is compiling boost from scratch. 

Now with the new windows terminal I believe we will have UTF-8 support and powerful toolchain, might not need mingw.

**Comment by AndrewSav** [comment]
On linux precompiled boost binaies comes via you default package manager for OS, so you do not need to build them. This is not the case for windows.

Also if you do not use appveryor / travis / other cloud build solution (because of the time limit) you need to get Dev enviroment running first, that is Windows Installation and Visual Studio. 

Installing Visual Studion in automated manner could be problematic depending on internet connection at where you are installing it to and from availabilty of the packages it needs to download during installation. Which means that if you are NOT using a cloud build sevice it is _very_ problematic.

**Comment by AndrewSav** [comment]
Now, that we have github actions this should be doable.
